run_localGPT.py

```diff
@@ -109,6 +109,11 @@ def main(device_type, model_type, openai_token):
         res = qa({"query": query})
         answer, docs = res['result'], res['source_documents']
 
+        if "question:" in answer:
```

Comment:
Why don't you use list instead of if-elif expression?
I think we will add lots of force stopping keywords.